doc/reference/repository-structure.md

```diff
@@ -78,14 +85,27 @@ rust/
 │  ├─ src/
 │  ├─ tests/
 │  ├─ Cargo.toml
-├─ e2e/
+├─ azure_iot_operations_services/
+│  ├─ examples/
+│  ├─ src/
+│  ├─ tests/
+│  ├─ Cargo.toml
+├─ azure_iot_operations_connector/
+│  ├─ examples/
+│  ├─ src/
+│  ├─ tests/
+│  ├─ Cargo.toml
+├─ sample_applications/
```

Comment:
This is an interesting file that seems easy to get out of date - instead of having the exact file structure, should we just have general guidelines/examples of the file structure so that this isn't a file that needs to be maintained as closely?

Reply:
I removed the specific samples from Rust. I could do so for Go as well if @clecompt-msft agrees